Tests/UnitTests/Mocks/MockIdentityAPI.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  MockIdentityAPI.swift
+//
+//  Created by Joshua Liebowitz on 6/16/22.
+
+import Foundation
+@testable import RevenueCat
+
+class MockIdentityAPI: IdentityAPI {
+
+    var invokedLogIn = false
+    var invokedLogInCount = 0
+    var invokedLogInParameters: (currentAppUserID: String, newAppUserID: String)?
+    var invokedLogInParametersList = [(currentAppUserID: String, newAppUserID: String)]()
+    var stubbedLogInCompletionResult: Result<(info: CustomerInfo, created: Bool), BackendError>?
```

Comment:
Maybe move these on top of the `logIn` method? I think other mocks are formatted like that so every method is next to these values.